### PR TITLE
Several improvements to sailthru content load

### PR DESCRIPTION
**Pull request by PaulWattenberger**

### CODE REVIEW

File: sailthru_content/sailthru_content.py [hunk 1]
```diff
@@ -0,0 +1,317 @@
...
+    """ Setup logging.
+
+    Add support for console logging, and avoid truncation by pandas.
+    """
+    msg_format = '%(asctime)s - %(levelname)s - %(message)s'
+    ch = logging.StreamHandler()
+    ch.setFormatter(logging.Formatter(msg_format))
+    ch.setLevel(logging.DEBUG)
+    logger.addHandler(ch)
+    logger.setLevel(logging.DEBUG)
```
Comment (cpennington) [inline sailthru_content/sailthru_content.py]: I'm not sure, but I think you could use https://docs.python.org/2/library/logging.html#logging.basicConfig to shorten this up a bit.

File: sailthru_content/sailthru_content.py [hunk 2]
```diff
@@ -0,0 +1,317 @@
...
+            response = sc.api_get('content', {'items': 1000})
+
+            if not response.is_ok():
+                logger.error("Error code %d connecting to Sailthru content api: %s",
+                             response.json['error'], response.json['errormsg'])
+                return
+
+            for body in response.json['content']:
+                logger.info(body)
+            return
```
Comment (cpennington) [inline sailthru_content/sailthru_content.py]: This `return` means that you only run through the `while True` loop once. Is that just a temporary debugging measure?

**Comment by cpennington** [comment]
A high-level comment: These scripts would likely be easier to read/review if `main` was split into some smaller more focused functions. I often have a function for specifying the argument parser, and then a function of each of the main script tasks (`list`, `clear` and `load` in this case).

**Comment by cpennington** [comment]
It would be nice to have https://travis-ci.org/ run the tests automatically, since you've written them.

File: sailthru_content/sailthru_content.py (hunk 2 above)
Comment (PaulWattenberger) [inline sailthru_content/sailthru_content.py]: Actually, the Sailthru content API doesn't have paging.  I'll get rid of the while True, it's confusing

File: sailthru_content/sailthru_content.py (hunk 1 above)
Comment (PaulWattenberger) [inline sailthru_content/sailthru_content.py]: I was trying to be consistent with other scripts

File: requirements.txt [hunk 3]
```diff
@@ -1,2 +1,3 @@
 MySQL-python==1.2.5
 pandas==0.16.2
+sailthru-client
```
Comment (clintonb) [inline requirements.txt]: Add a version. Also, given the divergent natures of these scripts, each script directory should have its own requirements file. There's no reason you should install either of the existing requirements for this script.

Comment (clintonb) [inline requirements.txt]: My comment was not addressed. :(

File: sailthru_content/sailthru_content.py (hunk 1 above)
Comment (clintonb) [inline sailthru_content/sailthru_content.py]: As the writer of those other scripts, I don't recommend trying to be consistent with them. They could probably use some TLC.

File: requirements.txt (hunk 3 above)
Comment (PaulWattenberger) [inline requirements.txt]: Patience is a rare and valuable virtue, though not one I possess myself....

**Pull request merged**